cli/azd/cmd/root.go

```diff
@@ -30,25 +31,37 @@ func NewRootCmd(staticHelp bool, middlewareChain []*actions.MiddlewareRegistrati
 	opts := &internal.GlobalCommandOptions{GenerateStaticHelp: staticHelp}
 	opts.EnableTelemetry = telemetry.IsTelemetryEnabled()
 
-	rootCmd := &cobra.Command{
-		Use:   "azd",
-		Short: "Azure Developer CLI is a command-line interface for developers who build Azure solutions.",
-		//nolint:lll
-		Long: `Azure Developer CLI is a command-line interface for developers who build Azure solutions.
+	productName := "Azure Developer CLI"
+	if opts.GenerateStaticHelp {
+		productName = "Azure Developer CLI (`azd`)"
+	}
+
+	shortDescription := heredoc.Docf(`%s is a command-line interface for developers who build Azure solutions.`, productName)
 
-To begin working with Azure Developer CLI, run the ` + output.WithBackticks("azd up") + ` command by supplying a sample template in an empty directory:
+	synopsisHeading := shortDescription + "\n\n"
+	if opts.GenerateStaticHelp {
+		synopsisHeading = ""
+	}
```

Comment:
Confused about this.. We build up the heading and then simply omit it here? Maybe we should restructure this?

Reply:
When generating static help the short description is displayed and then the long description. For clarity it was removed from the root level command. The same repetition does happen later in the documentation, however. 

Because this was an editorial decision we should revisit discussing this when we do the UX work. If we carry it forward it would probably best belong in `docgen.go`

Reply:
https://github.com/Azure/azure-dev/issues/1362